src/common/services/api/designations.service.js

```diff
@@ -255,6 +255,26 @@ class DesignationsService {
         return givingLinks
       })
   }
+
+  ministriesList (pagePath) {
+    const path = pagePath + '/jcr:content/content-parsys/designation_search_r.json'
+    return Observable.from(this.$http.get(this.envService.read('apiUrl') + '/' + path))
+      .map((response) => {
+        const ministries = []
+        if (response.data.ministries) {
+            angular.forEach(response.data.ministries, (v, k) => {
+              const m = JSON.parse(v)
```

Comment:
Not a big fan of single letter variable names.

Reply:
Done